### PR TITLE
Merge Hello-microverse-project to main 

### PR DESCRIPTION
**Pull request by den-cloud**
In this repository I created three files ( html, CSS, test.md ) to illustrate the use case of stylelint code analyzer and the gitignore file usage.

The html file includes an H1 heading with the text "Hello Microverse" which is styled from within the CSS file with a red color. Installed and configured are webhint and stylelint dependencies.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Nodejs
+
+### Setup
+- Run git clone `https://github.com/den-cloud/HelloMircoverse.git`
+- `cd HTML`
```
Comment (Emmyn5600) [inline README.md]: - [ ] Is it cd `HTML` or cd `HelloMicroverse`, please double check  to fix that issue.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+test.md
```
Comment (Emmyn5600) [inline .gitignore]: - [ ] Please add the node_module inside the .gitignore file as you did for the test.md

File: README.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+## Author
+
+👤 **Denzel Thandolwenkosi Nkomo**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
```
Comment (Emmyn5600) [inline README.md]: - [ ] Please replace the following with your accounts handle.

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @den-cloud  

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:. 
   
Changes required :writing_hand:  

 * #### Please check the comments under the review too
     
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED ✔✔**
Hello @den-cloud 👋🏽
Your project is complete!  There is nothing else to say. It's time to merge it.
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**